config.json

```diff
@@ -24,6 +24,13 @@
       "label": "<Create an empty project (no template)>",
       "path": "projects/no-template",
       "insertPath": "./"
+    },
+    {
+      "name": "elevate",
+      "label": "Elevate theme",
+      "path": "@hubspot/elevate",
+      "insertPath": "./src",
+      "isHubSpotAsset": true
```

Comment:
Should we keep the no-template option at the bottom? Does order here affects the order they get displayed?

Reply:
+1 on keeping no-template at the bottom